bob-apps/bin/bobd.rs

```diff
@@ -55,10 +55,16 @@ async fn main() {
     info!("Start backend");
     bob.run_backend().await.unwrap();
     info!("Start API server");
-    let http_api_port = matches
+    let http_api_port = if let Some(port) = matches
         .value_of("http_api_port")
         .and_then(|v| v.parse().ok())
-        .expect("expect http_api_port port");
+    {
+        port
+    } else if let Some(port) = node.http_api_port() {
```

Comment:
You can use .or(node.http_api_port()) in the first if condition. And IMO it's better to factor out `matches.value_of("http_api_port").and_then(|v| v.parse().ok())` into some variable, like `requestedPort` or smth else

Reply:
I set default `http_api_port` value in config to 8000 and replace `if` with `unwrap_or`